### DISCUSSION TITLE
Can you have more than one subdomain on GitHub Pages?

### DISCUSSION THREAD

**Discussion by SuperZekes** [community/community#116798] [Actions] [answered]
### Select Topic Area

Question

### Body

So I have [3dcube.superzekes.com](https://3dcube.superzekes.com) as a domain on GitHub Pages with my DNS records setup like this:

![image](https://github.com/community/community/assets/149198602/2c87e167-3bc0-4c52-9f52-c1d30e9f9e80)

Would I be able to have example.superzekes.com added to another one of my projects? Or is it just one subdomain at a time.

Best regards - 
@SuperZekes

**Comment by gil-son** [comment, marked as answer]
Hello,

Absolutely, it's possible

You need to create more CNAME. In this [doc](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain) you will find step by step

 If I supported you, consider to mark my answer. God bless you!

**Reply by SuperZekes** [reply]
You're so kind thank you and this works perfectly! It will come in handy when I make a wiki website for my next project!
